* deno's upgrade feature cannot work on OpenBSD, directly disable it
* disable deno_panic
* disable vsock

Index: cli/Cargo.toml
--- cli/Cargo.toml.orig
+++ cli/Cargo.toml
@@ -32,7 +32,7 @@ harness = false
 path = "./bench/lsp_bench_standalone.rs"
 
 [features]
-default = ["upgrade", "__vendored_zlib_ng"]
+default = ["__vendored_zlib_ng"]
 # A feature that enables heap profiling with dhat on Linux.
 # 1. Compile with `cargo build --profile=release-with-debug --features=dhat-heap`
 # 2. Run the executable. It will output a dhat-heap.json file.
@@ -49,7 +49,7 @@ hmr = ["deno_runtime/hmr", "deno_snapshots/disable"]
 __vendored_zlib_ng = ["flate2/zlib-ng-compat", "libz-sys/zlib-ng"]
 lsp-tracing = ["dep:tracing", "dep:tracing-subscriber", "dep:tracing-opentelemetry"]
 # Custom panic trace with `deno_panic`
-panic-trace = ["deno_panic"]
+# panic-trace = ["deno_panic"]
 
 [build-dependencies]
 deno_runtime = { workspace = true, features = ["include_js_files_for_snapshotting", "only_snapshotted_js_sources"] }
@@ -192,6 +192,7 @@ winapi = { workspace = true, features = ["knownfolders
 [target.'cfg(unix)'.dependencies]
 nix.workspace = true
 shell-escape = "=0.1.5"
+[target.'cfg(any(target_os = "linux", target_os = "macos"))'.dependencies]
 tokio-vsock.workspace = true
 
 [dev-dependencies]
